### PR TITLE
Added functionality for running sysbench

### PR DESCRIPTION
**Pull request by acalhounRH**
### Description
In order to support bare metal testing using sysbench+benchmark-Wrapper we added the ability to trigger and evaluate expected output for ES indexing. 

@jtaleric 
@dry923

### CODE REVIEW

File: snafu/sysbench/trigger_sysbench.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import subprocess
+import logging
+
+#  import json
```
Comment (jtaleric) [inline snafu/sysbench/trigger_sysbench.py:17]: old?

File: snafu/sysbench/trigger_sysbench.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+        sample_starttime = datetime.utcnow().strftime("%s")
+        # run the sysbench test
+        stdout, stderr, rc = self._run_sysbench()
+        #  setup result summary dictionary
+        sysbench_result_summary = {
+            "uuid": self.uuid,
+            "user": self.user,
+            "clustername": self.cluster_name,
+            "Date": sample_starttime,
```
Comment (jtaleric) [inline snafu/sysbench/trigger_sysbench.py:85]: lowercase

File: snafu/sysbench/trigger_sysbench.py (hunk 1 above)
Comment (acalhounRH) [reply snafu/sysbench/trigger_sysbench.py]: removed.

File: snafu/sysbench/trigger_sysbench.py (hunk 2 above)
Comment (acalhounRH) [reply snafu/sysbench/trigger_sysbench.py]: changed

**Comment by acalhounRH** [comment]
Looks like the testing host may have ran out of disk space, there are 14 Errors similar to the below one. 

Error Summary
19:39:08 -------------
19:39:08 Disk Requirements:
19:39:08    At least 1MB more space needed on the / filesystem.

**Review by dry923 — APPROVED** [review]
LGTM!

**Comment by mohit-sheth** [comment]
LGTM

**Comment by jtaleric** [comment]
lol pgbench_wrapper random failure.

**Pull request merged**